Makefile

```diff
@@ -84,8 +84,8 @@ endif
 ifeq (${ENABLE_JAVA_SDK}, ON)
 	cd $(WORKING_DIR)/analytical_engine/java && \
 	mvn clean install -DskipTests && \
-	sudo cp ${WORKING_DIR}/analytical_engine/java/graphscope-runtime/target/native/libgrapee-jni.so ${INSTALL_PREFIX}/lib/ && \
-	sudo cp ${WORKING_DIR}/analytical_engine/java/graphscope-runtime/target/graphscope-runtime-0.1-shaded.jar ${INSTALL_PREFIX}/lib/ && \
+	sudo cp ${WORKING_DIR}/analytical_engine/java/grape-runtime/target/native/libgrape-jni.so ${INSTALL_PREFIX}/lib/ && \
+	sudo cp ${WORKING_DIR}/analytical_engine/java/grape-runtime/target/grape-runtime-0.1-shaded.jar ${INSTALL_PREFIX}/lib/ && \
```

Comment:
Are we going to rename the target to `grape`?

Reply:
I have renamed all `graphscope` in directory/artifact into `grape`, since now our java sdk will be in directory `core/analytical/java`, because as we are renaming our Java SDK to `grape-jdk`(pointed out by @yecol ), does it seem strange to  have `graphscope-*` stuff under a `grape-jdk` project?

Reply:
OK fair enough. Thanks for clarifying.